src/components/carousel/__tests__/index.spec.js

```diff
@@ -0,0 +1,83 @@
+import {map} from 'lodash';
+import React from 'react';
+import {render, fireEvent} from '@testing-library/react-native';
+import {Text, View} from 'react-native';
+import {Constants} from '../../../helpers';
+import Carousel from '../index';
+
+const numberOfPagesShown = 5;
+const eventData = {
+  nativeEvent: {
+    contentOffset: {
+      x: Constants.screenWidth
+    }
+  }
+};
+const onChangePageMock = jest.fn();
+const onScrollMock = jest.fn();
+const props = {
+  testID: 'carousel',
+  initialPage: 0,
+  pagingEnabled: true,
+  autoplay: false,
+  autoplayInterval: 4000,
+  horizontal: true,
+  onChangePage: onChangePageMock,
+  onScroll: onScrollMock
+  //animatedScrollOffset: // set to check Animated
+};
+
+const TestCase = props => {
+  return (
+    <Carousel {...props}>
+      {map([...Array(numberOfPagesShown)], (_, index) => (
+        <Page key={index}>
+          <Text testID={`page-${index}`}>Page #{index}</Text>
+        </Page>
+      ))}
+    </Carousel>
+  );
+};
+
+const Page = ({children, ...others}) => {
+  return (
+    <View {...others} style={{flex: 1}}>
+      {children}
+    </View>
+  );
+};
+
+describe('Carousel render tests', () => {
+  
+  describe('default setup', () => {
+    it('should be set to default', () => {
```

Comment:
Please make the description more explicit

Reply:
Done